### DISCUSSION TITLE
CMU 15-131 计算机科学家的伟大实用工具 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#50] [Announcements]
# CMU 15-131 计算机科学家的伟大实用工具 | HackWay技术学习路线

课程名称： Great Practical Ideas for Computer Scientists

https://hackway.org/docs/cs/freshman/tools/cs15131

**Comment by hzy029** [comment]
怎么没有视频，悲
